business/tls.go

```diff
@@ -115,9 +114,62 @@ func (in *TLSService) NamespaceWidemTLSStatus(ctx context.Context, namespace, cl
 	return models.MTLSStatus{
 		Status:          mtlsStatus.NamespaceMtlsStatus(namespace).OverallStatus,
 		AutoMTLSEnabled: mtlsStatus.AutoMtlsEnabled,
+		Cluster:         cluster,
+		Namespace:       namespace,
 	}, nil
 }
 
+func (in *TLSService) ClusterWideNSmTLSStatus(ctx context.Context, nss []string, cluster string) ([]models.MTLSStatus, error) {
```

Comment:
Is this limited to the mTLS status of the passed in namespaces? If so then this might be better named something like `CrossNamespacemTLSStatus` since it's not scoped to the whole cluster just the provided namespaces.

Reply:
'CrossNamespacemTLSStatus' seems to me also a bit confusing, as when namespaces are not provided the method loads all namespaces from provided cluster by default. I will leave it as it is before some better idea will come. The idea of having `ClusterWide...` in names is having this task https://github.com/kiali/kiali/issues/4832 in a mind, to switch Namespace related fetches to Cluster related ones.